src/NzbDrone.Core.Test/ParserTests/ParserFixture.cs

```diff
@@ -66,6 +66,13 @@ public void should_parse_quality_from_extension(string title)
             Parser.Parser.ParseAlbumTitle(title).Quality.QualitySource.Should().Be(QualitySource.Extension);
         }
 
+        [TestCase("of Montreal-Hissing Fauna, Are You The Destroyer? 2007", "Hissing Fauna, Are You The Destroyer")]
+        [TestCase("of Montreal - 2007 - Hissing Fauna, Are You The Destroyer?", "Hissing Fauna, Are You The Destroyer")]
+        public void should_parse_album(string title, string correct)
+        {
+            Parser.Parser.ParseAlbumTitle(title).AlbumTitle.Should().Be(correct);
```

Comment:
Can you integrate artist check and release year check into this test to ensure it's parsing out everything correctly